src/main/resources/application.properties

```diff
@@ -11,4 +11,7 @@ spring.jpa.database-platform = org.hibernate.dialect.MySQL8Dialect
 spring.jpa.hibernate.naming.physical-strategy = org.hibernate.boot.model.naming.PhysicalNamingStrategyStandardImpl
 
 # --- SERVER CONFIGURATION ---
-server.port = 8080
+server.port = 8443
+server.ssl.key-store = classpath:${KEYSTORE_PATH}
```

Comment:
Consider removing `classpath:`, since many people tend to install SSL certificates outside the application directory, and it won't be compiled with the application files.

Reply:
Done